R/utils.R

```diff
@@ -0,0 +1,13 @@
+# if a Sb Directory doesn't exist it Creates one (in the working directory) and moves
+# the wd to the sub directory. It returns the original wd
+
+createDir = function(sub.dir) {
```

Comment:
please insert comment to add relative path as subdirectory, instead of C:/User/../currentWD/subdirectory just "subdirectory" just to be very sure